themes/classic/templates/_partials/head.tpl

```diff
@@ -43,10 +43,26 @@
     <link rel="canonical" href="{$page.canonical}">
   {/if}
   {block name='head_hreflang'}
-      {foreach from=$urls.alternative_langs item=pageUrl key=code}
-            <link rel="alternate" href="{$pageUrl}" hreflang="{$code}">
-      {/foreach}
+    {foreach from=$urls.alternative_langs item=pageUrl key=code}
+      <link rel="alternate" href="{$pageUrl}" hreflang="{$code}">
+    {/foreach}
   {/block}
+  
+  {block name='head_microdata'}
+    {include file="_partials/microdata/head-jsonld.tpl"}
+  {/block}
+  
+  {block name='head_microdata_special'}{/block}
+  
+  {block name='head_pagination_seo'}
+    {include file="_partials/pagination-seo.tpl"}
+  {/block}
+
+  <meta property="og:title" content="{$page.meta.title}" />
```

Comment:
I didn't catch it earlier but it would be cool to have it override possible as well, could you add block here @fdonnet ?

maybe 

`{block name='head_open_graph'}`

Reply:
yes, perfect, done.